src/clue/app-config.json

```diff
@@ -62,8 +62,8 @@
           {
             "title": "Group Workspaces",
             "type": "problem-documents",
-            "dataTestHeader": "group-workspaces",
-            "dataTestItem": "group-list-items",
+            "dataTestHeader": "left-group-workspaces",
```

Comment:
As we discussed separately, with the removal of the right nav tabs we probably don't need these to be prefixed with `left-` any more.

Reply:
Agreed. Fixed.